azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/middleware/sdk/ChatServiceListener.kt

```diff
@@ -66,4 +78,56 @@ internal class ChatServiceListener(
             dispatch(ChatAction.AllMessagesFetched())
         }
     }
+
+    private fun handleInfoModel(it: ChatEventModel) {
+        when (it.infoModel) {
+            is MessageInfoModel -> {
+                when (it.eventType) {
+                    ChatEventType.CHAT_MESSAGE_RECEIVED -> {
+                        val model = it.infoModel
+                    }
+                    ChatEventType.CHAT_MESSAGE_EDITED -> {
+                        val model = it.infoModel
+                    }
+                    ChatEventType.CHAT_MESSAGE_DELETED -> {
+                        val model = it.infoModel
+                    }
+                    else -> {}
+                }
+            }
+            is ParticipantTimestampInfoModel -> {
+                when (it.eventType) {
+                    ChatEventType.TYPING_INDICATOR_RECEIVED -> {
+                        val model = it.infoModel
+                    }
+                    ChatEventType.READ_RECEIPT_RECEIVED -> {
+                        val model = it.infoModel
+                    }
+                    else -> {}
+                }
+            }
+            is ChatThreadInfoModel -> {
+                when (it.eventType) {
+                    ChatEventType.CHAT_THREAD_DELETED -> {
+                        val model = it
+                    }
+                    ChatEventType.CHAT_THREAD_PROPERTIES_UPDATED -> {
+                        val model = it
+                    }
+                    else -> {}
+                }
+            }
+            is RemoteParticipantsInfoModel -> {
+                when (it.eventType) {
+                    ChatEventType.PARTICIPANTS_ADDED -> {
+                        val model = it
+                    }
+                    ChatEventType.PARTICIPANTS_REMOVED -> {
+                        val model = it
+                    }
+                    else -> {}
+                }
+            }
```

Comment:
Lots of duplicated code here. Would it make sense to combine the cases with the same outcome (use a `,` to join the cases with the same code)?

Reply:
In future, different code will be added here :)